tooling/gatsby-plugin/src/theme.js

```diff
@@ -1,3 +1,3 @@
-import { theme } from "@chakra-ui/react"
+import { theme, baseTheme } from "@chakra-ui/react"
 
-export default theme
+export { theme, baseTheme }
```

Comment:
Is this not a breaking change??

My gatsby dev would not work until I changed the theme.js overrider from a default export to a named export??

Reply:
Hey @natac13! Sorry for the trouble and thank you for bringing up this error.

We have updated it in #7149 to get the original default `theme` object to be a default export within the plugin. We have not noticed any issues from here after some testing. 

Let us know from there if you receive further trouble!